monitor/services/src/main/java/org/datacleaner/monitor/scheduling/SchedulingService.java

```diff
@@ -42,7 +42,8 @@
 public interface SchedulingService extends RemoteService {
 
     @RolesAllowed({ SecurityRoles.VIEWER, SecurityRoles.SCHEDULE_EDITOR })
-    public List<ScheduleDefinition> getSchedules(TenantIdentifier tenant) throws DCSecurityException;
+    public List<ScheduleDefinition> getSchedules(TenantIdentifier tenant, boolean loadProperties) 
+            throws DCSecurityException;
```

Comment:
Can we keep the old method as well and just redirected it to this one with "getSchedules(tenant, true)" ? Maybe someone else uses the method.